src/Spice86.Core/Emulator/CPU/CPU.cs

```diff
@@ -1219,8 +1235,12 @@ private static bool IsStringOpUpdatingFlags(int stringOpCode)
     }
 
     private bool TestJumpConditionCXZ() {
```

Comment:
Shouldn't this be public so low-level C# overrides of ASM code can call it ?

Reply:
For generated code, you will know whether you are in 16 or 32 bits beforehand so you will either do CX ==0 or ECX == 0 but no if